fuzzypp/test/main.cpp

```diff
@@ -0,0 +1,4 @@
+#define CATCH_CONFIG_MAIN
+#include <catch.hpp>
+
+TEST_CASE("1: Test entrypoint", "main") {}
```

Comment:
I've never used catch before, so I'm curious what this test does?

Reply:
Catch will automatically generate a `main` method for us (see L1). I believe we have to add a dummy test here to get everything to work for reasons unknown. :smile:

Reply:
👍